### DISCUSSION TITLE
What are the advantages Dotmin.Sync has over MySQL replication ?

### DISCUSSION THREAD

**Discussion by stlsw** [Mimetis/Dotmim.Sync#569] [Q&A]
I don't see any advantages Dotmin.Sync can provide except it is net core library.  We are in the middle of making a decision for syncing our merchant's database into cloud, our server is in net core 3.1. Please correct me if I am wrong.

**Comment by Mimetis** [comment]
It's not a **replication** system, but more a **synchronization** system, where you are sending only a subset of the data to the client (and from client to server)

It's an Hub And Spoke sync system: **One Server** and **Multiples Clients**
**Conflicts** are handled, and can be manually resolved.
**Filters** can be used to filter client specific datas

Clients can be any kind of relational databases, especially `SQLite` if you are developing a mobile application. 
You are not limited to `MySQL` to `MySQL`

It's working across HTTP(S) connection, through a Web API exposed.

Honestly, if you have to sync `MySQL` databases **only**, that are reachable from each other, I guess going for `MySQL` replication is the way to go, since it's an out of the box solution

But if you  have to create a sync solution involving mobile applications for example, `DMS` can be a solution.

Make sense ?

**Reply by stlsw** [reply]
You are right. It make a lot of sense for SQLite.
